src/components/ContributorIndicators.tsx

```diff
@@ -2,10 +2,10 @@ import MailOutlineIcon from '@mui/icons-material/MailOutline';
 import { Box, IconButton, Tooltip } from '@mui/material';
 import { useTranslation } from 'react-i18next';
 import OrcidLogo from '../resources/images/orcid_logo.svg';
-import { Contributor } from '../types/contributor.types';
+import { PreviewContributor } from '../types/contributor.types';
 
 interface ContributorIndicatorsProps {
-  contributor: Contributor;
+  contributor: PreviewContributor;
```

Comment:
Ser ut som at dette fører til at vi må gjør en del ekstra arbeid med å caste over til riktig type og sånn mange plasser, som virker unødig komplekst, da det vi egentlig trenger her bare er info om `orcid` og `correspondingAuthor`. Med dette får vi fjernet en del unødvendig konvertering


```suggestion
  orcid?: string;
  correspondingAuthor?: boolean;
```